avm/src/vm/protos/mod.rs

```diff
@@ -1 +1,3 @@
-// Dummy mod. Will be auto generated by protoc-rust
```

Comment:
Hmmm  will think how to avoid this and be transparent for us

Reply:
sounds good. I brought it back in the next commit, but I keep forgetting the 1 extra step after `git add .`